src/Symfony/Component/Notifier/Bridge/LightSms/LightSmsTransport.php

```diff
@@ -0,0 +1,171 @@
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Vasilij Duško <vasilij@prado.lt>
+ */
+final class LightSmsTransport extends AbstractTransport
+{
+    protected const HOST = 'www.lightsms.com';
+
+    private $login;
+    private $password;
+    private $from;
+
+    private const ERROR_CODES = [
+        1 => 'Missing Signature',
+        2 => 'Login not specified',
+        3 => 'Text not specified',
+        4 => 'Phone number not specified',
+        5 => 'Sender not specified',
+        6 => 'Invalid signature',
+        7 => 'Invalid login',
+        8 => 'Invalid sender name',
+        9 => 'Sender name not registered',
+        10 => 'Sender name not approved',
+        11 => 'There are forbidden words in the text',
+        12 => 'Error in SMS sending',
+        13 => 'Phone number is in the blackist. SMS sending to this number is forbidden.',
+        14 => 'There are more than 50 numbers in the request',
+        15 => 'List not specified',
+        16 => 'Invalid phone number',
+        17 => 'SMS ID not specified',
+        18 => 'Status not obtained',
+        19 => 'Empty response',
+        20 => 'The number already exists',
+        21 => 'No name',
+        22 => 'Template already exists',
+        23 => 'Missing Month (Format: YYYY-MM)',
+        24 => 'Timestamp not specified',
+        25 => 'Error in access to the list',
+        26 => 'There are no numbers in the list',
+        27 => 'No valid numbers',
+        28 => 'Missing start date (Format: YYYY-MM-DD)',
+        29 => 'Missing end date (Format: YYYY-MM-DD)',
+        30 => 'No date (format: YYYY-MM-DD)',
+        31 => 'Closing direction to the user',
+        32 => 'Not enough money',
+        33 => 'Missing phone number',
+        34 => 'Phone is in stop list',
+        35 => 'Not enough money',
+        36 => 'Can not obtain information about phone',
+        37 => 'Base Id is not set',
+        38 => 'Phone number already exists in this database',
+        39 => 'Phone number does not exist in this database',
+        999 => 'Unknown Error',
+    ];
+
+    public function __construct(string $login, string $password, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->login = $login;
+        $this->password = $password;
+        $this->from = $from;
+
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        return sprintf('lightsms://%s/external/get/send.php?from=%s', $this->getEndpoint(), $this->from);
```

Comment:
```suggestion
        return sprintf('lightsms://%s?from=%s', $this->getEndpoint(), $this->from);
```

This should be removed IMHO